### ISSUE TITLE
About the version of your caffe

### ISSUE THREAD

**Issue by CuriousCat-7** [wangzheallen/transfer_ijcv#1]
Hi, 
I am trying to convert your caffe model to torch and pytorch.
But the well-used caffe2torch converter(loadcaffe) does not recognize the BN layer. And other converters do not recognize the parameters(frozen, slop filter, etc) in batch normalization layers.

So I am wondering does this project use official caffe code? Actually the BN layer of the .prototext files cannot be recognized by http://ethereon.github.io/netscope/#/editor, a well-known caffe visualization online tool.

So, could you please updata the .prototxt and .caffemodel in official caffe? 
Or, are there any pre-trained model written in other frameworks?

Thank you 
Best,
Neo

**Comment by wanglimin** [comment]
These models are trained using our modified caffe toolbox:

https://github.com/yjxiong/caffe
